<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Page_Records</class>
 <widget class="QWidget" name="Page_Records">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>我的病例</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>就诊时间</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="listWidget"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="4">
     <widget class="QLabel" name="doctor_and_department">
      <property name="text">
       <string>医生: null  |  科室: null</string>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed"/>
      </property>
     </widget>
    </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>症状:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="3">
    <widget class="QLabel" name="sympptoms">
     <property name="text">
      <string>null</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="maximumHeight">
      <number>80</number>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed"/>
     </property>
    </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>医嘱:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="3">
    <widget class="QTextEdit" name="prescription">
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding"/>
     </property>
     <property name="minimumHeight">
      <number>80</number>
     </property>
     <property name="maximumHeight">
      <number>200</number>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
    </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
